src/main/java/com/okta/tools/helpers/ConfigHelper.java

```diff
@@ -0,0 +1,56 @@
+package com.okta.tools.helpers;
+
+import com.okta.tools.aws.settings.Configuration;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Reader;
+
+public final class ConfigHelper {
+    /**
+     * Gets a reader for the config file. If the file doesn't exist, it creates it
+     *
+     * @return The file reader for the config file
+     * @throws FileNotFoundException
+     * @author Andrei Hava
+     * @since 02/14/2018
+     */
+    public static Reader getConfigReader() throws FileNotFoundException {
+        return FileHelper.getReader(FileHelper.getAwsDirectory().toString() + "/config");
+    }
+
+    /**
+     * Gets a FileWriter for the config file
+     *
+     * @return The FileWriter for the config file
+     * @throws IOException
+     * @author Andrei Hava
+     * @since 02/14/2018
+     */
+    public static FileWriter getConfigWriter() throws IOException {
+        return FileHelper.getWriter(FileHelper.getAwsDirectory().toString() + "/config");
```

Comment:
:bug: It's unsafe to build paths with String concatenation.

Recommendation: move the pre-existing Path-based methods from OktaAwsCliAssumeRole.